train.py

```
@@ -57,3 +71,95 @@
+                    step_time / (step + 1) * iter_per_epoch // 60 - step_time // 60))
+                if (wandb is not None):
+                    wandb.log({"loss": loss.item() * train_cfg.grad_accumulation_steps,
+                           "lr": optimizer.param_groups[-1]['lr'],
+                           "epoch_Time": step_time / (step + 1) * iter_per_epoch // 60 - step_time // 60})
+            
+            # Validation
+            if (step + 1) % train_cfg.validation_interval == 0:
+                model.eval()
+                with torch.no_grad():
+                    val_loss = 0
+                    for val_step, val_batch in enumerate(val_data_loader):
```

Ruff (0.8.2)

Loop control variable `val_step` not used within loop body

(B007)